From 7c7331440d22b1e987d0e6fd0d4337e57011e6cc Mon Sep 17 00:00:00 2001
From: Dominique Leuenberger <dimstar@opensuse.org>
Date: Mon, 14 Nov 2016 12:51:11 +0100
Subject: [PATCH] Include libgupnp/gupnp-context-manager.h

As we make use of the type 'GUPnPContextManager' we need to ensure that
this is known here. Relying on any other random header to being this in
for us is unreliable (and has been seen failing in the wild).
---
 libdleyna/server/upnp.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libdleyna/server/upnp.h b/libdleyna/server/upnp.h
index 6e1aa556c473..ca053fc53eae 100755
--- a/libdleyna/server/upnp.h
+++ b/libdleyna/server/upnp.h
@@ -24,6 +24,7 @@
 #define DLS_UPNP_H__
 
 #include <libdleyna/core/connector.h>
+#include <libgupnp/gupnp-context-manager.h>
 
 #include "client.h"
 #include "async.h"
-- 
2.9.3

